src/allotropy/parsers/beckman_vi_cell_blu/vi_cell_blu_parser.py

```diff
@@ -62,33 +63,34 @@ def __init__(self, column_name: str, data_type: Any) -> None:
         self.data_type: Any = data_type
 
 
-class _Sample(NamedTuple):
+@dataclass(frozen=True)
+class _Sample:
     data_frame: pd.DataFrame
     row: int
 
-
-def _get_value(sample: _Sample, column: str) -> Optional[Any]:
-    data_frame, row = sample
-    if column not in data_frame.columns:
-        return None
-    return data_frame[column][row]
-
-
-def _get_value_not_none(sample: _Sample, column: str) -> Any:
-    value = _get_value(sample, column)
-    if value is None:
-        msg = f"Missing value for column '{column}'."
-        raise AllotropeConversionError(msg)
-    return value
+    def get_value(self, column: str) -> Optional[Any]:
+        if column not in self.data_frame.columns:
+            return None
+        return self.data_frame[column][self.row]
+
+    def get_value_not_none(self, column: str) -> Any:
```

Comment:
Any reason for this to not be a boolean flag on `get_value`. I dont have a strong feeling just curious as to your thoughts

Reply:
It could be, but I try to limit my refactors to doing one thing at a time. Moving functions to methods and changing their signatures is two things :)